docker-compose-mac.yml

```diff
@@ -68,7 +68,7 @@ services:
     ports: ["8000:80"]
     environment:
       OSRD_DEV: "True"
-      OSRD_BACKEND_URL: "http://osrd-core"
+      OSRD_BACKEND_URL: "http://host.docker.internal:8080"
```

Comment:
Why did you have to change it?

Reply:
It didn't work before when I ran it on my windows laptop